frontend/src/components/pages/events/EventDetails/index.tsx

```diff
@@ -0,0 +1,232 @@
+import { useMutation, useQuery } from "@apollo/client";
```

Comment:
Have a question about why you added 
`event.attendable.deadline && dayjs(event.attendable.deadline).isAfter(dayjs()) && CountdownButton`
deadline is as of now not a required field for an attendable event.

Reply:
Probably a poor assumption on our part. We had no restriction preventing users from signing up after the deadline. Suggest changing this to something related to either event start or deadline passed.

Reply:
Not allowing signups after the deadline is already handled within the CountDownButton, but updated it so that it uses the start time if no deadline is provided. So I do not think it is needed to do checks before rendering the CountDownButton as well.

Reply:
Sound great!